apps/studio/src/features/editing-experience/components/CreateFolderModal/CreateFolderModal.tsx

```diff
@@ -45,7 +44,26 @@ export const CreateFolderModal = ({
   siteId,
   parentFolderId,
 }: CreateFolderModalProps): JSX.Element => {
-  const { setValue, register, handleSubmit, watch, formState, getFieldState } =
+  return (
+    <Modal isOpen={isOpen} onClose={onClose}>
+      <ModalOverlay />
+      <CreateFolderModalContent
```

Comment:
same for this one, key should be here instead of inside the component. If not rerender may be one step behind.

Reply:
done